README

```diff
@@ -16,3 +16,5 @@ several of them using the Restructured Text markup notation.
 Please read the Documentation/process/changes.rst file, as it contains the
 requirements for building and running the kernel, and information about
 the problems which may result by upgrading your kernel.
+
+See the file CI.md for a description of continuous integration.
```

Comment:
@Asiderr This should be removed as CI.md is no longer present

Reply:
@Asiderr

Reply:
@macpijan fixed